sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -951,7 +957,8 @@ number
     | MINUS? BIGDECIMAL_LITERAL       #bigDecimalLiteral
     ;
 
-// When `spark.sql.dialect.spark.ansi.enabled=true`, there are 2 kinds of keywords in Spark SQL.
+// When we use PostgreSQL dialect or use Spark dialect with
+// `spark.sql.dialect.spark.ansi.enabled=true`, there are 2 kinds of keywords in Spark SQL.
```

Comment:
We can simply say "when use_SQL_standard_keywords=true". The parser doesn't care when it is true, we should explain it in the caller side.

Reply:
Copy that, done in f16b6ea.